addon/components/one-way-input.js

```diff
@@ -86,12 +86,16 @@ const OneWayInputComponent = Component.extend({
   },
 
   init() {
-    this._super(...arguments);
```

Comment:
Well, apparently I fail at computers for the day.  I guess this is ok as long as you leave a comment explaining why (`// @rwjblue sucks at releasing and 2.0.3 contains a dumb bug`)...